core/src/test/java12/com/linecorp/armeria/server/JavaHttpClientUpgradeTest.java

```diff
@@ -73,6 +75,31 @@ protected void configure(ServerBuilder sb) {
     @BeforeEach
     void setup() {
         serverChannelPipeline.set(null);
+
+        bodyPublisher = new BodyPublisher() {
```

Comment:
Can you revert unrelated changes in this PR? I prefer that flaky tests be handled separately - I think it's fine if some flaky tests don't pass in the CI run

Reply:
Oh I see. Thanks.